### DISCUSSION TITLE
Import and other options menus greyed

### DISCUSSION THREAD

**Discussion by JParcoeur** [freeplane/freeplane#467] [Help] [answered]
Most of the time when I try to Import the options box has everything greyed out (trying to to import a Mindmanager .mmap file).  I had it working a couple of times but have no idea how. The Help dropdown menu also seems to grey out on me on occasion. I am a newbie with Freeplane so it might be something obvious. Would some give me clue on how to fix this?
Thanks!
PC running Windows 10.

**Comment by abc16361** [comment, marked as answer]
Maybe your map is on 'Viewing'. To edit a map, it needs to set it on 'Editing' (menu Edit -> Editing).

**Reply by JParcoeur** [reply]
Yep, you are right. I thought it would be something basic like that. Thank you!!!!!
